contracts/token/types/Token6.sol

```diff
@@ -196,22 +145,8 @@ library Token6Lib {
      * @param account Account to check
      * @return Token balance of the account
      */
-    function balanceOf(Token6 self, address account) internal view returns (UFixed18) {
-        return fromTokenAmount(IERC20(Token6.unwrap(self)).balanceOf(account));
-    }
-
-    /**
-     * @notice Converts the unsigned fixed-decimal amount into the token amount according to
-     *         it's defined decimals
-     * @dev Provides the ability to "round up" the token amount which is useful in situations where
-     *      are swapping one token for another and don't want to give away "free" units due to rounding
-     *      errors in the favor of the user.
-     * @param amount Amount to convert
-     * @param roundUp Whether to round decimal token amount up to the next unit
-     * @return Normalized token amount
-     */
-    function toTokenAmount(UFixed18 amount, bool roundUp) private pure returns (uint256) {
-        return roundUp ? Math.ceilDiv(UFixed18.unwrap(amount), OFFSET) : UFixed18.unwrap(amount) / OFFSET;
+    function balanceOf(Token6 self, address account) internal view returns (UFixed6) {
+        return UFixed6.wrap(IERC20(Token6.unwrap(self)).balanceOf(account));
```

Comment:
nit: you can delete `function fromTokenAmount()` as well

Reply:
good catch!